core/domain/exp_beam_job.py

```diff
@@ -0,0 +1,91 @@
+# limitations under the License.
+
+"""Beam Jobs for exploration models"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.domain import exp_fetchers
+from core.platform import models
+from jobs import base_jobs
+from jobs import job_utils
+from jobs.io import ndb_io
+from jobs.types import job_run_result
+
+import apache_beam as beam
+
+(exp_models,) = models.Registry.import_models( # type: ignore[no-untyped-call]
+    [models.NAMES.exploration])
+
+
+class PopulateExplorationWithProtoSize(base_jobs.JobBase):
+    """Update the proto_size_in_bytes in exploration models."""
+
+    def run(self): # type: ignore[no-untyped-def]
+        """Update every exploration model in datastore."""
+
+        exploration_model_query = exp_models.ExplorationModel.query()
+        migrated_models = (
+            self.pipeline
+            | 'Get every Exploration Model' >> (
+                ndb_io.GetModels(  # type: ignore[no-untyped-call]
+                    exploration_model_query, self.datastoreio_stub
+                ))
+            | 'Update Exploration Model' >>
+            beam.ParDo(UpdateProtoSizeInBytes()))
+
+        # Save the successfully migrated models.
+        _ = migrated_models | ndb_io.PutMulti() # type: ignore[attr-defined]
+        success_message = (
+            migrated_models
+            | beam.combiners.Count.Globally()
+            | beam.Map(lambda count: '%d models sucessfully stored')
+            | beam.Map(job_run_result.JobRunResult.as_stdout)
+        )
+
+        error_messages = (
+            migrated_models.error
+            | beam.Map(job_run_result.JobRunResult.as_stderr)
+        )
+
+        return (success_message, error_messages) | beam.Flatten()
+
+
+class UpdateProtoSizeInBytes(beam.DoFn): # type: ignore[misc]
+    """Calculate proto size and return updated model."""
+
+    def process(self, input_model): # type: ignore[no-untyped-def]
+        """Update each exploration model."""
+
+        if input_model.proto_size_in_bytes == 0:
+            model = job_utils.clone_model( # type: ignore[no-untyped-call]
+                input_model)
+            try:
+                # Fetching corresponding domain object here to run the
+                # validate() function on it.
+                old_exploration = exp_fetchers.get_exploration_from_model( # type: ignore[no-untyped-call]
+                    model)
+                old_exploration.validate()
+                # Calculate the proto size.
+                model.proto_size_in_bytes = 1
+            except Exception as e:
+                error_message = (
+                    'Exploration Model %s failed to update'
+                    'proto_size_in_bytes: %s' % (model.id, e))
+                yield beam.pvalue.TaggedOutput('error', error_message)
+            else:
+                yield model
```

Comment:
Hi @brianrodri 
Is the Beam jobs tab in the release coordinator page is ready to use? 
I tried adding one of my beam jobs to oppia/jobs/registry.py and checked in the Beam jobs tab, the list is empty. Am I missing any step here?
I removed the job entry in the register right now in the PR.

Reply:
@vojtechjelinek

Reply:
No it is not ready yet, Beam jobs can only be tested in the backend.

Reply:
resolving as beam jobs are in now.